doc/source/train/getting-started-pytorch-lightning.rst

```diff
@@ -3,7 +3,7 @@
 Getting Started with PyTorch Lightning
 ======================================
 
-This tutorial will walk you through the process of converting an existing PyTorch Lightning script to use Ray Train.
+This tutorial walks through the process of converting an existing PyTorch Lightning script to use Ray Train.
```

Comment:
Can you apply the same changes to the other getting started guides that follow the same format (Torch, Transformers)?

Reply:
Done!